workspace/__lib__/xod/core/pulse-on-change(byte)/patch.cpp

```diff
@@ -0,0 +1,16 @@
+
+struct State {
+    Number sample = NAN;
```

Comment:
```suggestion
    uint8_t sample = 0;
```

I'd change the `sample` type to `uint8_t` to save a little bit of memory.

Everything else looks good 👍🏻